### PR TITLE
Draft RFP-013

### PR DESCRIPTION
**Pull request by asilinphoto**
Draft while pending additional information for RFP-013. Additional reviewers to be added once we have more content

### CODE REVIEW

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Funds can be used to support graduate students, the summer salaries of university faculty, travel to conferences and meetings, participation in summer schools and Web3 events, fellowships for graduate students and postdocs. 
+
+## Scope of this RFP
+
+The goal of this RFP is to fund research that addresses one or more of the open problems above while meeting the requirements/constraints of the overall project and driving progress towards the goals. Proposals should seek to maintain compatibility with portions of the solution that they do not replace or improve upon.
+
+## RFP Details
+
+### Application deadline
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **July 30, 2023**, or earlier if the allocated funding is exhausted.
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Recommended team
+ 
+
+### Award
+Commensurate with scope, up to USD 35k per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
Commensurate with scope, up to $35,000 per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000. 
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
+
+### Recommended team
+ 
+
+### Award
+Commensurate with scope, up to USD 35k per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000. 
+
+### Payout schedule
+60% upon award and 40% on completion (adjustable to accommodate institutional requirements).
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
Paid upon award.
```

There's no point in incurring the overhead of two payments for a $25,000 grant.

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+### Point of contact
+**(TAG Rosario)**. We encourage you to reach out to research-grants@protocol.ai or visit #lobby in the Cryptonet Discord **(LINK?)** if you are considering applying or have any questions.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
@rosariogennaro. We encourage you to reach out to research-grants@protocol.ai or visit #lobby in the Cryptonet Discord **(LINK?)** if you are considering applying or have any questions.
```

Comment (jsoares) [reply RFPs/DRAFT-rfp-013-cryptonet-network-grant]: Is this discord public? Just making sure before we post a link.

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# RFP-013: Cryptonet Network Grant
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]: Are we calling them network grants or microgrants? The cryptonet website says the latter. I prefer the former, given these aren't microgrants (e.g. per FIL's definition, up to $5k).

@rosariogennaro ?

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- Fully Homomorphic Encryption
+- Security against Rational Adversaries
+- Quantum Resistant Cryptography
+- Proofs of Space
+
+Funds can be used to support graduate students, the summer salaries of university faculty, travel to conferences and meetings, participation in summer schools and Web3 events, fellowships for graduate students and postdocs. 
+
+## Scope of this RFP
+
+The goal of this RFP is to fund research that addresses one or more of the open problems above while meeting the requirements/constraints of the overall project and driving progress towards the goals. Proposals should seek to maintain compatibility with portions of the solution that they do not replace or improve upon.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## RFP Details
+
+### Application deadline
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
+
+### Recommended team
+ 
+
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant]:
```suggestion
```

**Comment by jsoares** [comment]
And I fixed the filename when reviewing so all of my comments are now outdated --; Will review again...

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+Funds can be used to support graduate students, the summer salaries of university faculty, travel to conferences and meetings, participation in summer schools and Web3 events, fellowships for graduate students and postdocs. 
+
+## Scope of this RFP
+
+The goal of this RFP is to fund research that addresses one or more of the open problems above while meeting the requirements/constraints of the overall project and driving progress towards the goals. Proposals should seek to maintain compatibility with portions of the solution that they do not replace or improve upon.
+
+## RFP Details
+
+### Application deadline
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **July 30, 2023**, or earlier if the allocated funding is exhausted.
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+### Recommended team
+ 
+
+### Award
+Commensurate with scope, up to USD 35k per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
Commensurate with scope, up to $35,000 per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000. 
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
+
+### Recommended team
+ 
+
+### Award
+Commensurate with scope, up to USD 35k per award. We expect the typical grant to last one year (with possible renewal) and be funded on average $25,000. 
+
+### Payout schedule
+60% upon award and 40% on completion (adjustable to accommodate institutional requirements).
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
Paid upon award.
```

There's no point in incurring the overhead of two payments for a $25,000 grant.

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+### Point of contact
+**(TAG Rosario)**. We encourage you to reach out to research-grants@protocol.ai or visit #lobby in the Cryptonet Discord **(LINK?)** if you are considering applying or have any questions.
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
@rosariogennaro. We encourage you to reach out to research-grants@protocol.ai or visit #lobby in the Cryptonet Discord **(LINK?)** if you are considering applying or have any questions.
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
+# RFP-013: Cryptonet Network Grant
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]: Are we calling them network grants or microgrants? The cryptonet website says the latter. I prefer the former, given these aren't microgrants (e.g. per FIL's definition, up to $5k).

@rosariogennaro ?

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+- Security against Rational Adversaries
+- Quantum Resistant Cryptography
+- Proofs of Space
+
+Funds can be used to support graduate students, the summer salaries of university faculty, travel to conferences and meetings, participation in summer schools and Web3 events, fellowships for graduate students and postdocs. 
+
+## Scope of this RFP
+
+The goal of this RFP is to fund research that addresses one or more of the open problems above while meeting the requirements/constraints of the overall project and driving progress towards the goals. Proposals should seek to maintain compatibility with portions of the solution that they do not replace or improve upon.
+
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+## RFP Details
+
+### Application deadline
+**Rolling**: we will be reviewing applications in batches corresponding to calendar months. The call will close on **September 30, 2022**, or earlier if awarded.
+
+### Recommended team
+ 
+
```
Comment (jsoares) [inline RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]:
```suggestion
```

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant (hunk 5 above)
Comment (jsoares) [reply RFPs/DRAFT-rfp-013-cryptonet-network-grant]: Also note that the cryptonet page still mentions the exception for postdocs, which we agreed to remove. We should either update the website and sync back, or finalise this and push the changes to the website.

File: RFPs/DRAFT-rfp-013-cryptonet-network-grant.md (hunk 12 above)
Comment (jsoares) [reply RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]: Also still pending, this passage in the Cryptonet website:
>An exception to the budget limit will be made for postdoc applications.

Comment (rosariogennaro) [reply RFPs/DRAFT-rfp-013-cryptonet-network-grant.md]: Reverted to Network Grants and removed the mention of "exception". Cryptonet Website will be updated

**Review by jsoares — APPROVED** [review]

**Pull request merged**